node/Cargo.toml

```diff
@@ -92,9 +92,9 @@ cumulus-relay-chain-minimal-node = { git = "https://github.com/paritytech/cumulu
 cumulus-relay-chain-rpc-interface = { git = 'https://github.com/paritytech/cumulus.git', branch = "polkadot-v0.9.37" }
 
 # Nimbus dependencies
-nimbus-consensus = { git = "https://github.com/manta-network/nimbus.git", tag = "v4.0.7" }
-nimbus-primitives = { git = "https://github.com/manta-network/nimbus.git", tag = "v4.0.7" }
-pallet-author-inherent = { git = "https://github.com/manta-network/nimbus.git", tag = "v4.0.7" }
+nimbus-consensus = { git = "https://github.com/manta-network/nimbus.git", tag = "v4.0.8" }
```

Comment:
no need to upgrade the nimbus tag, so far we've only been upgrading it when there are any changes on the nimbus side

Reply:
Right, i just pushed the same commit on the new tag for consistency.
can there be any problems with that ? I want to avoid restarting the CI

Reply:
branch structure on nimbus looks weird, let me doublecheck that the tagged commit is actually ok

Reply:
ok, looks good